entity-framework/core/performance/advanced-performance-topics.md

```diff
@@ -109,3 +109,16 @@ Even if the sub-millisecond difference seems small, keep in mind that the consta
 
 > [!NOTE]
 > Avoid constructing queries with the expression tree API unless you really need to. Aside from the API's complexity, it's very easy to inadvertently cause significant performance issues when using them.
+
+## Reducing runtime overhead
+
+As with any layer, EF Core adds a bit of runtime overhead compared to coding directly against lower-level database APIs. This runtime overhead is unlikely to impact most real-world applications in a significant way; the other topics in this performance guide, such as query efficiency, index usage and minimizing roundtrips, are far more important. In addition, even for highly-optimized applications, network latency and database I/O will usually dominate any time spent inside EF Core itself. However, for high-performance, low-latency applications where every bit of perf is important, the following recommendations can be used to reduce EF Core overhead to a minimum:
+
+* Turn on [DbContext pooling](#dbcontext-pooling); our benchmarks show that this feature can have a decisive impact on high-perf, low-latency applications.
+  * Make sure that the `maxPoolSize` corresponds to your usage scenario; if it is too low, DbContext instances will be constantly created and disposed, degrading performance. Setting it too high may needlessly consume memory as unused DbContext instances are maintained in the pool.
+  * For an extra tiny perf boost, consider using <xref:Microsoft.EntityFrameworkCore.Infrastructure.PooledDbContextFactory%601> instead of having DI inject context instances directly (EF Core 6 and above). DI management of DbContext pooling incurs a slight overhead.
+* Use compiled queries for hot queries.
```

Comment:
"precompiled"?

Reply:
Sure. We don't have documentation for that feature yet, so it has no name :)

Will change to precompiled.